specifyweb/frontend/js_src/lib/components/QueryBuilder/Fields.tsx

```diff
@@ -81,6 +81,20 @@ export function QueryFields({
       distance: 4,
     });
 
+    sortable.on('sortable:start', (event) => {
+      const target = event.dragEvent.originalEvent.target as HTMLElement;
+      if (
+        target.closest('button') !== null ||
+        target.closest('a') !== null ||
+        target.closest('input') !== null ||
+        target.closest('select') !== null ||
+        target.closest('textarea') !== null ||
+        target.closest('article') !== null
```

Comment:
'article' can be used for several things, not just by combo box. if you want to match combo box, replace `article` with `.custom-select`